introduction.md

```diff
@@ -0,0 +1 @@
+Hi! I am Jesús López and I really want to learn how to use git properly. I'm a musician who aims to create a new music learning app.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.